installation/gcp/dm/network-template.py

```diff
@@ -0,0 +1,31 @@
+# Copyright 2016 Google Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Creates the network."""
+
+
+def GenerateConfig(context):
+  """Creates the network with environment variables."""
+
+  resources = [{
+      'name': context.env['name'],
```

Comment:
Is that mean that the "cloud NAT" and the network will have the same name? (both using context.env['name'])

Reply:
context.env['name'] is the name passed into the template from the config/wrapper template. For example the wrapper vmclarity template uses this network template like:

```
resources [
   { 
       name: vmclarity-{context.env['deployment']}-network
       type: network-template.py
    }
 ]
 ```
 
context.env['name'] becomes `vmclarity-{context.env['deployment']}-network` in this case.